Bottom: 7434181e1760631d2de50fd4381ffd51b6fc8c38
Top:    d01783e69f8ae9c8a1e0dd7e92f6fbb422ff9ca4
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-12-31 17:56:04 -0500

Refresh of zarlink

---

diff --git a/.gitignore b/.gitignore
index 1806af2..0392deb 100644
--- a/.gitignore
+++ b/.gitignore
@@ -24,3 +24,4 @@
 TAGS*~
 Packages
 Packages.gz
+patches*
